src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java

```diff
@@ -25,7 +25,7 @@ public static void emptyListError() {
     }
 
     public static void deleteFormatError() {
-        printer.showToUser("Wrong format, please enter in the format: delete [move ID]");
+        printer.showToUser("Wrong format, please enter in the format:\n\t delete [move ID]");
```

Comment:
For consistency in printing, can change here to "delete [MOVE_ID]"

Reply:
Good idea